dotcom-rendering/src/web/components/Card/Card.tsx

```diff
@@ -57,6 +58,8 @@ export type Props = {
 	// Labs
 	branding?: Branding;
 	supportingContent?: DCRSupportingContent[];
+	type?: string;
+	enriched?: DCREnrichedContent;
```

Comment:
This feels like the wrong place to do this. I'm not super familiar with what a 'snap' is though but is it really a type of `Card`?